### PR TITLE
EO-JavaScript-Quiz

### PR DESCRIPTION
**Pull request by eorstrom**
I was able to get all of the code working except for the addEventListener on the 'enter' keypress, which I have commented out toward the bottom of the quiz.js file.

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  } 
+    // Create treeObject to hold height and character properties
+    treeObject = {
+      height: treeHeight,
+      character: treeCharacter
+  };
+}
+
+// Make a function that loops through treeHeight and logs each row to the console
+var growTree = function () {
```
Comment (chortlehoort) [inline quiz.js]: Missing requirement:

Create a tree function that should build a pine tree out of a character in the browser console.. **It accepts a single object as an argument.**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<body>
+
+    <h2>Grow a Tree!</h2>
+        <p>
+            <label>Height of your tree? (please enter a number)<input id ="treeHeight" type="text"></label>
+        </p>
+        <p>
+            <label>Character to use to build your tree? (for example: * or #)<input id="treeCharacter" type="text"></label>
+        </p>
+        <button id="buildTree-btn">Build My Tree!</button>
```
Comment (chortlehoort) [inline index.html]: Requirement: a button that is labeled "Grow your tree"

File: quiz.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    var treeHeightVal = parseInt(treeObject.height)
+        console.log(treeHeightVal);
+    var charInput = treeObject.character;
+        console.log(charInput);
+    var treeRow;
+
+    // For loop that loops through the number of times indicated by the treeHeightVal variable and logs each treeRow to the console each time it's looped through
+    for (var i = 1; i <= treeHeightVal; i++) {
+        treeRow = "";
+        treeRow += ' '.repeat(treeHeightVal - i) + charInput.repeat((2 * i) - 1);
```
Comment (chortlehoort) [inline quiz.js]: Can you elaborate on what this line of code is doing?

**Comment by chortlehoort** [comment]
@eorstrom Great job. Just a couple of small things that I commented on. Your first reaction may be "Really?" to them, but it's critical for when you're out there professionally that you follow requirements on a software project.

Fix those up, and provide the elaboration I asked about, and we'll get this to complete.

**Comment by JoeShep** [comment]
Looks great so far. You do need to add the listener for when the user hits enter/return, but otherwise it's in great shape.

**Comment by C-Stein** [comment]
Nice work! You've met all the requirements of the assignment. The only tweak you might consider making is when the alert pops up, the function still creates a "ghost tree" of empty lines in the console. :+1:

**Pull request merged**